### PR TITLE
Temporary CI metadata/haproxy fix

### PR DESCRIPTION
**Pull request by wangxian-ibm**
CI has been having intermittent metadata issue and it's caused by haproxy not started properly. Restart haproxy before CI tests to temporarily fix the issue.

### CODE REVIEW

**Comment by wangxian-ibm** [comment]
retest

**Comment by wangxian-ibm** [comment]
retest

**Comment by nirajdp76** [comment]
retest

File: playbooks/tests/tasks/all.yml [hunk 1]
```diff
@@ -13,6 +13,14 @@
       msg: "OS is {{ ansible_distribution }}"
     failed_when: ursula_os is not defined
 
+# bandaid metadata issue
+- name: restart haproxy service
+  hosts: controller
+  tasks:
+  - service:
+      name: haproxy.service
```
Comment (nirajdp76) [inline playbooks/tests/tasks/all.yml]: this will not work for ubuntu. Might want to try 
service:
  name: haproxy

Comment (wangxian-ibm) [inline playbooks/tests/tasks/all.yml]: oops
updated, thanks :)

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest - just to see how ci holds

**Pull request merged**